go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/bitcomplete/sqltestutil
 go 1.18
 
 require (
-	github.com/docker/docker v20.10.16+incompatible
+	github.com/docker/docker v24.0.0+incompatible
```

Comment:
I think this update introduces the requirement to use an empty opts for the stop container call. This is why I updated it here.